src/app.js

```diff
@@ -25,6 +25,10 @@ var OffsetToLineColumnConverter = require('./lib/offsetToLineColumnConverter')
 
 var examples = require('./app/example-contracts')
 
+// components
+var fileExplorer = require('_file-explorer')
```

Comment:
Not sure if we should make those changes that are really related to the internal architecture of the app right now in this PR.
Actually I don't really like the idea of having this manually manage `node_module`  with prefixed name, it looks like mixing dependencies and internal module. But ok that is very nice to require a module without having to remember the full path :)
Do we need to have this `package.json` file in the file-explorer folder? what is the benefit of having it?
If the reason of doing that is only because of the easier use of `require`,we should find another way to require internal class in a easiest way.
So merging this could be fine for me (only as a temporary approach of requiring internal stuff), we should think on a better way (if there is).